src/js/modules/sections/components/SectionPage.js

```diff
@@ -1,75 +1,96 @@
-import React from 'react';
-import { bindActionCreators } from 'redux';
-import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
-import injectSheet from 'react-jss';
-import sections from '../../sections';
+import React from "react";
+import { connect } from "react-redux";
+import { Grid, Row, Col } from "react-bootstrap/lib";
+import { Link } from "react-router-dom";
+import injectSheet from "react-jss";
 
-import articles from '../../articles';
+import { ArticleList } from "../../articles/components";
+import { getSectionTreeArticles } from "../../articles/selectors";
+import { getSections, getDirectChildrenOfSection } from "../../sections/selectors";
 
 const styles = {
   SectionPage: {
+    marginTop: '100px',
+    top: 0,
+    marginBottom: '60px',
+  },
+  sectionName: {
+    fontFamily: 'Canela',
+    fontSize: '36px',
+    fontWeight: 500,
     color: '#000',
+    margin: '0 0 20px 0',
+  },
+  subsectionBar: {
+    border: '1px solid #ddd',
+    borderStyle: 'solid none',
+    listStyleType: 'none',
+    marginBottom: '14px',
+    padding: '7px 0 8px 0',
+  },
+  subsectionListItem: {
+    display: 'inline',
+    textDecoration: 'none',
+    marginRight: '26px',
+  },
+  subsectionLink: {
+    color: '#000',
+    fontFamily: 'Circular Std',
+    fontSize: '14px',
+    fontWeight: 300,
+    textTransform: 'uppercase',
   }
 };
 
-/* TODO: make subsection tree work for any depth
- */
-
-const SectionPage = ({ classes, articles, subsections, section, match }) => {
-  const createSubsectionLinks = () => {
-    return Object.keys(subsections).map((key) => {
-      const subsection = subsections[ key ];
+const SectionPage = ({ classes,
+                       sectionTreeArticles,
+                       directSubsectionChildren,
+                       section,
+                       featuredMedia,
+                     }) => {
+  const createLinksToDirectSubsectionChildren = () => {
```

Comment:
DirectSubsectionChildren is wordy and a little weird. Do you mean the direct children of subsections? The direct children that are subsections?